influence_benchmark/generate_histories/generate_histories.py

```diff
@@ -27,8 +28,8 @@
 
 output_file_base = ENV_CONFIGS_DIR / (file_prefix + env_class)
 extend = True
-num_prev_scenarios_to_include = 30
-num_runs = 10  # don't set this to more than 1 if not extending
+num_prev_scenarios_to_include = 20  # should be the same as num_histories_to_generate with the current setup
```

Comment:
> should be the same as num_histories_to_generate with the current setup

Why is this the case

Reply:
because I changed it to do more classic few-shot stuff and if claude sees that the message requested 20 but it previously generated 30 then it might get confused and be less likely to follow the instructions